app/views/pages/tutorials/notifications/creating-api-call-notification.liquid

```diff
@@ -47,13 +50,31 @@ headers: '{
 {% endraw %}
 ```
 
+This defines a POST request to the endpoint with the newly created user's ID. In this example the body of the request is JSON, and to notify the endpoint about it, you set the `Content-Type` header to `application/json`. If the call is successful, then the code from `callback` property is triggered. You can access endpoint's response via [Response](/api-reference/graphql/objects#response) object - which includes headers, status and body. In the example we assume the server returns valid JSON, which we parse and log.
```

Comment:
I think adding link to https://documentation.platformos.com/tutorials/forms/form-server-side-processing might help to propagate this knowledge.